src/main/scala/se/nimsa/sbx/user/UserServiceActor.scala

```diff
@@ -111,25 +112,34 @@ class UserServiceActor(userDao: UserDAO, superUser: String, superPassword: Strin
 
   def userByName(name: String): Future[Option[ApiUser]] = userDao.userByName(name)
 
-  def createOrUpdateSession(user: ApiUser, ip: String, userAgent: String): Future[ApiSession] =
-    userDao.userSessionByUserIdIpAndUserAgent(user.id, ip, userAgent)
+  def createOrUpdateSession(user: ApiUser, ip: String, userAgent: String): Future[ApiSession] = {
+    val userAgentHash = md5Hash(userAgent)
+
```

Comment:
User agent is now digested to an md5 hash to fit the limited mysql column length.